src/main/java/mate/academy/springboot/swagger/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.swagger.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
```suggestion
@Data
```
Lilia mentioned, that it is better not to use Data annotation with entities: https://habr.com/ru/company/haulmont/blog/564682/